circle.yml

```diff
@@ -0,0 +1,11 @@
+## Customize dependencies
+dependencies:
+  override:
+    - npm install -g grunt-cli
+    - npm install grunt
+    - npm install grunt-contrib-qunit
+
+## Customize test commands
+test:
+  override:
+    - grunt test
```

Comment:
it prints something like: 
>> 38 assertions passed (106ms)

Done, without errors.
It is not anything more than what is already there